LambdaEngine/Include/Game/ConsoleCommand.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+namespace LambdaEngine
+{
+	
+
+	struct LAMBDA_API Arg
+	{
+		enum EType { FLOAT, BOOL, INT, STRING, EMPTY };
```

Comment:
Is there a reason for this not being an enum class?

Reply:
no